scripts/post-process.py

```diff
@@ -46,13 +47,9 @@
 
         # ensure integrity in nix sri format
         if not hashStr.endswith("=") or "-" not in hashStr:
-            result = subprocess.run(
-                ["nix-hash", "--to-sri", "--type", hashAlgo, hashStr],
-                text=True,
-                encoding="ascii",
-                stdout=subprocess.PIPE,
+            hashesToNormalize.append(
```

Comment:
FYI, this was actually a workaround because of the missing `builtins.hashTo` function. This is however now fixed: [issue](https://github.com/NixOS/nix/issues/3151) and [MR](https://github.com/NixOS/nix/pull/7708)

This means we could set the SRI hash during the Nix evaluation. This could improve evaluation time since no processes would need to be spawned.

Note this is an improvement and doesn't need to be part of this MR.